src/testFixtures/java/pl/edu/pw/elka/paprykaisalami/geeruh/support/StatusAttributeDataset.java

```diff
@@ -0,0 +1,12 @@
+package pl.edu.pw.elka.paprykaisalami.geeruh.support;
+
+public class StatusAttributeDataset {
+
+    public static final String FIRST_STATUS_CODE = "FST";
```

Comment:
NTH: I would change it to something else so that it doesn't get mistaken with project code when debugging tests. It was weird for me to see
```
[
  {
    "historicIssue": {
      "issueId": "FST-3",
      "statusCode": "FST",
      "type": "TASK",
      "summary": "First issue",
      "description": "Important issue"
    },
    "timestamp": "2022-12-13T16:33:41.554+00:00",
    "type": "INSERT"
  },
  {
    "historicIssue": {
      "issueId": "FST-3",
      "statusCode": "FST",
      "type": "BUG",
      "summary": "Second issue",
      "description": "Even more important issue"
    },
    "timestamp": "2022-12-13T16:37:23.039+00:00",
    "type": "UPDATE"
  }
]
```

Reply:
Good catch, I forgot to change it 😄